Basic/2DSpaceShooter/Assets/Scripts/ShipControl.cs

```diff
@@ -134,6 +137,8 @@ void Start()
     {
         DontDestroyOnLoad(gameObject);
         SetPlayerUIVisibility(true);
+        shootAction = InputSystem.actions.FindAction("Fire");
+        moveAction = InputSystem.actions.FindAction("Move");
```

Comment:
is that possible? 
Are the actions drag & droppable? 👀

Reply:
Yes! You can use an `InputActionReference` instead of the InputAction directly.
Each action in the map is imported as a sub-asset with a deterministic Id. Renaming the action or moving it around in the map will maintain existing references to it.

Reply:
Thanks for the hint, intrestign! I used serialized fields now, is this what you meant?
Or should I somehow use something like [InputControl(layout = "Keyboard"), SerializeField]?
Not sure how exactly that one works though? It is not happy about the layout "Keyboard".